### ISSUE TITLE
If you navigate the “Drop-Down” box using “Arrow Key”, labeling of combo box is not readable

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3657]
**Reported by kalidas on 2013-11-21 13:42**
If use navigate the “Drop-Down” box using “Arrow Key”, labeling of combo box is not readable by using  Firefox 25.0.1 and NVDA 2013.1.1
Could you please suggest .

**Comment by ehollig** [comment]
Closing, as this information is not clear enough, as reported in https://github.com/nvaccess/nvda/issues/3657#issuecomment-155312899
